tests/src/Functional/Lock/RedisLockFunctionalTest.php

```
@@ -31,10 +31,13 @@ protected function setUp() {
     $filename = $this->siteDirectory . '/settings.php';
     chmod($filename, 0666);
     $contents = file_get_contents($filename);
+    $redis_interface = getenv('REDIS_INTERFACE');
```

nice, great to see that this is working so well. Maybe we should have a default or at least throw a clear exception if this value is not set. I see there are a few setup methods, we could possibly make that a simple trait and throw the exception there, the trait could maybe just return the value in one method and another that sets the settings.